coral/src/app/features/topics/request/TopicRequest.tsx

```diff
@@ -148,16 +140,16 @@ function TopicRequest() {
               )}
               required={true}
             />
-            <Box component={Flexbox} gap={"l1"}>
-              <Box component={FlexboxItem} grow={1} width={"1/2"}>
+            <Box component={Box.Flex} gap={"l1"}>
+              <Box component={Box.Flex} grow={1} width={"1/2"}>
```

Comment:
This does not render the same layout as before (the `width` rule is not applied):

<img width="954" alt="Screenshot 2023-09-29 at 09 21 58" src="https://github.com/Aiven-Open/klaw/assets/20607294/f01238f6-aa72-4389-8e9f-3c2c29f8213a">


 Additionally, the way the `<Box component={}` pattern is not recommended, and is at odds with the standard way we do things elsewhere.
 
 This allows to preserve the layout, and is much more straightforward:
 
 ```tsx
<Box.Flex gap={"l1"}>
  <Box width={"1/2"}>
    <SelectOrNumberInput
      name={"topicpartitions"}
      label={"Topic partitions"}
      max={selectedEnvironment?.params?.maxPartitions}
      required={true}
    />
  </Box>
  <Box width={"1/2"}>
    <SelectOrNumberInput
      name={"replicationfactor"}
      label={"Replication factor"}
      max={selectedEnvironment?.params?.maxRepFactor}
      required={true}
    />
  </Box>
</Box.Flex>
 ```
 
 Renders this:
 
<img width="908" alt="Screenshot 2023-09-29 at 09 18 42" src="https://github.com/Aiven-Open/klaw/assets/20607294/7fcdc322-4cab-4bc7-bb6a-9bdddb7e5e88">